RELEASE-NOTES.txt

```diff
@@ -35,6 +35,7 @@
 - Prevents weird effects in live previews due to incomplete input [#1822](https://github.com/Automattic/simplenote-electron/pull/1822)
 - Fixed a bug where searching for a tag containing non-alphanumeric characters erroneously returned no notes [#1828](https://github.com/Automattic/simplenote-electron/pull/1828)
 - Fixed a bug causing the search result highlighting in the sidebar to be case sensitive [#1831](https://github.com/Automattic/simplenote-electron/pull/1831)
+- Properly close revision/history view when clicking outside of slider [#1837](https://github.com/Automattic/simplenote-electron/pull/1837)
```

Comment:
Looks like I added this to 1.15 above in #1837 so we need to fix that in `develop` and then not add it here in this branch. I'll quick-update `develop` to prevent the merge issue.